src/swagger-specs/segmentation.yaml

```diff
@@ -3041,16 +3020,20 @@ components:
             sandboxId:
               type: string
               example: 6ed34f6f-fe21-4a30-934f-6ffe21fa3075
+              description: The ID of the sandbox.
             sandboxName:
               type: string
               example: prod
+              description: The name of the sandbox.
             type:
               type: string
               example: production
+              description: The type of the sandbox.
```

Comment:
Can we list the possible values here? I'm assuming it can be either `production` or `development`

Reply:
I'd assume so as well.